src/pod-security-policy/capabilities/src_test.rego

```diff
@@ -207,6 +227,27 @@ test_input_drop_undefined_x2 {
    results := violation with input as input
    count(results) == 2
 }
+test_input_drop_literal_all {
+   input := { "review": input_init_review([cdrop(["ALL"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 0
+}
+test_input_drop_literal_all_lower {
+   input := { "review": input_init_review([cdrop(["all"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 0
+}
+test_input_drop_literal_all_with_all_param {
```

Comment:
can we add a test for test_input_drop_literal_with_all_param where dropped does not include `all` and count(results) > 0?

Reply:
@ritazh added the test